beaker-vue/src/pages/NotebookInterface.vue

```diff
@@ -90,17 +89,34 @@
             </SideMenu>
         </template>
         <template #right-panel>
+            <SideMenu
+                position="right"
+                :show-label="true"
+                highlight="line"
+                :expanded="true"
+                initialWidth="25vi"
+                :maximized="isMaximized"
+            >
+                <SideMenuPanel label="Preview" icon="pi pi-eye" no-overflow>
+                    <PreviewPane :previewData="contextPreviewData"/>
+                </SideMenuPanel>
+                <SideMenuPanel id="file-contents" label="Contents" icon="pi pi-file" no-overflow>
+                    <PreviewPanel
```

Comment:
`PreviewPane` and `PreviewPanel` are confusing.
Mind moving `panels/PreviewPanel` to `panels/FileContentsPanel` and  `misc/PreviewPane` to `panels/PreviewPanel`?